indonesia/ceritabaru

```diff
@@ -0,0 +1,5 @@
+hi there, today i'll create story using indonesia language, check this out
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.